videointelligence/annotate/object_tracking_gcs_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package annotate
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestObjectTrackingGCS(t *testing.T) {
```

Comment:
This timed out after 25 minutes. Any way we can speed it up? Smaller file? If not, we should use `testutil.EndToEndTest(t)` instead of `SystemTest`, that way it only runs only one version nightly.

Reply:
```
=== RUN   TestObjectTrackingGCS
panic: test timed out after 25m0s
```

Reply:
The GCS version usually takes 30 seconds in my machine, with the occasional 1 to 2 minutes. My guess is that something hanged for this test, it shouldn't take that long to run.

Reply:
Interesting, running the test using the same command as kokoro (`GOCACHE=off time go test -timeout 25m -v . ./videointelligence/...`) makes the tests take longer and timeout for some reason.

Reply:
The `cat.mp4` video is only 6.1MB, so it should be small enough. In comparison, the `googlework_short.mp4` is 1.4MB. There seems to be some inconsistency in the time it takes the test to complete, so it might be a bit brittle.  I'm still testing some variations, but I'll probably end up marking the ObjectTracking tests to `EndToEndTest` so it will give it more time.